app/views/shared/_navbar.html.erb

```diff
@@ -26,7 +26,35 @@
           <%= link_to "Community", chat_link, class: 'nav-link', target: '_blank', rel: 'noreferrer' %>
         </li>
 
-        <li class="nav-link">
+        <% if cookies[:theme] == "dark" %>
+          <li class="nav-item nav-link" id="dark-mode-sun" style="flex-direction: row; justify-content: center; overflow:visible">
+            <%= link_to root_path(theme: "light"), data: { test_id: "dark-mode-button" }, class: "dark-mode-button sun" do %>
+              <svg xmlns="http://www.w3.org/2000/svg" class="h-6 w-6" fill="none" viewBox="0 0 24 24" stroke="currentColor" style="overflow:visible"> 
+                <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M12 3v1m0 16v1m9-9h-1M4 12H3m15.364 6.364l-.707-.707M6.343 6.343l-.707-.707m12.728 0l-.707.707M6.343 17.657l-.707.707M16 12a4 4 0 11-8 0 4 4 0 018 0z" />
+                <text x="15" y="5" font-size="10">
+                  <tspan baseline-shift = "super">
+                  BETA
```

Comment:
This being positioned right-aligned just at the top of the icons looks a little off to me. I think it would look better centred above. Or maybe we don't need it at all?

Design is subjective though, maybe we should get feedback from a few more people?

Reply:
The problem with centering above I found was that with the sun icon, there was some cutoff and pushing it higher made it seem a little weird. Agree on getting feedback from others. Since the server side stuff Austin added is so smooth, I think we don't need it.